app/views/practice/show.scala

```diff
@@ -17,7 +17,7 @@ object show:
       moreCss = cssTag("analyse.practice"),
       modules = List(analyseNvuiTag),
       pageModule = PageModule(
-        "analysisBoard.study",
```

Comment:
fun fact, back in the days I had to rename our `analysis`/`analyse` JS assets to `analysisBoard`, to prevent them from being filtered away by ad-blockers.

Reply:
We'll need to name the source file in ui/analyse analysisBoard.ts then due to the naming restrictions imposed by esbuild on content hashed assets. It's complicated why this is necessary but it is. That OK?

Reply:
ok let's try it as is